src/generic/FormSwitchGroup.jsx

```diff
@@ -37,20 +34,28 @@ export default function FormSwitchGroup({
             checked={checked}
           />
         </div>
+        <Form.Text
+          className="mt-0 pr-3"
+          id={helpTextId}
+          muted
+        >
+          {helpText}
+        </Form.Text>
       </div>
 
     </Form.Group>
   );
 }
 FormSwitchGroup.propTypes = {
   id: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
-  helpText: PropTypes.string.isRequired,
+  label: PropTypes.node.isRequired,
```

Comment:
`helpText` / `label` proptype is changed —> text to node —>  which is already in use by other components

Reply:
Text is a type of node, so this should not cause issues. This just allows us to use other react components for label, so you can have icons etc.